husky_components/datagrid/decorators/table-view.js

```diff
@@ -741,7 +730,7 @@ define(function() {
                     content,
                     column.type,
                     Object.keys(this.table.rows).length,
-                    record.hasOwnProperty('id') ? record['id'] : null
```

Comment:
perhaps we want to use the datagrid in content ... there we have `uuid` instead of `id`! can we make `idKey` configurable?